static/js/components/subject_page/block.tsx

```diff
@@ -52,10 +53,16 @@ export interface BlockPropType {
   svgChartHeight: number;
 }
 
+const NUM_TILES_SHOWN = 6; // for NL: show 2 rows (3 tiles per row).
+const HIDE_TILE_CLASS = "tile-hidden";
+
 export function Block(props: BlockPropType): JSX.Element {
   const columnWidth = props.columns
     ? `${(100 / props.columns.length).toFixed(2)}%`
     : "0";
+  // HACK for NL. Assumes all charts are in a single column.
+  const showExpando = isNlInterface();
+  const hideIfIndexGreater = showExpando ? NUM_TILES_SHOWN : -1;
```

Comment:
nit: minIdxToHide or something like that? 'hideIfIndexGreater' sounds like it should be a boolean of whether or not to hide.

Reply:
good call - done